src/main/java/com/wcc/platform/configuration/ObjectMapperConfig.java

```diff
@@ -50,14 +51,17 @@ private void registerCustomDeserializers(final ObjectMapper objectMapper) {
     final DateTimeFormatter formatter =
         DateTimeFormatter.ofPattern(DATE_TIME_FORMAT, Locale.ENGLISH);
 
-    final JavaTimeModule javaTimeModule = new JavaTimeModule();
-    javaTimeModule.addDeserializer(ZonedDateTime.class, new ZonedDateTimeDeserializer(formatter));
-    javaTimeModule.addSerializer(ZonedDateTime.class, new ZonedDateTimeSerializer(formatter));
-
     objectMapper
-        .registerModule(javaTimeModule)
+        .registerModule(
+            new JavaTimeModule()
+                .addDeserializer(ZonedDateTime.class, new ZonedDateTimeDeserializer(formatter))
+                .addSerializer(ZonedDateTime.class, new ZonedDateTimeSerializer(formatter)))
         .registerModule(
             new SimpleModule()
+                .addSerializer(ImageType.class, new LowerCaseEnumSerializer())
+                .addSerializer(ColorType.class, new LowerCaseEnumSerializer())
+                .addSerializer(ColorShadeType.class, new LowerCaseEnumSerializer())
+                .addSerializer(SocialNetworkType.class, new LowerCaseEnumSerializer())
```

Comment:
Do we need to add LowerCaseEnumSerializer for the MemberType as well?

Reply:
I don't think we need, because that should not be part of css classes in my opinion. But we can do it later if necessary.